src/Compilers/Core/MSBuildTask/Portable/BuildClientShim.cs

```diff
@@ -16,9 +16,6 @@ internal static class BuildClientShim
             BuildPaths buildPaths,
             string keepAlive,
             string libEnvVariable,
-            CancellationToken cancellationToken)
-        {
-            throw new NotSupportedException();
-        }
+            CancellationToken cancellationToken) => Task.FromResult<BuildResponse>(null);
```

Comment:
There's no catch around the calling code though -- this currently crashes the build.